lesson03/assignment/index.html

```diff
@@ -7,11 +7,15 @@
 	<title>Fun with Constructors!</title>
   <link rel="stylesheet" href="./css/style.css">
 	<script src="index.js"></script>
+	
 </head>
+
 <body>
 	<h1>Assignment 3</h1>
 	<section class="template-hook">
 		Fill this section with the templated components
+		<script src="index.js"></script>
```

Comment:
No need to call the script a second time.

Reply:
Often times, when a script will edit the dom, you can place it right before the closing </body> tag.